data/messages.proto

```diff
@@ -363,13 +363,26 @@ message LSPInformation {
   /// the channel.
   int64 min_htlc_msat = 11;
 
-  int64 channel_fee_permyriad = 12;
+  int64 channel_fee_permyriad = 12 [deprecated = true];
 
   bytes lsp_pubkey = 13;
   // The channel can be closed if not used this duration in seconds.
-  int64 max_inactive_duration = 14;
-
-  int64 channel_minimum_fee_msat = 15;
+  int64 max_inactive_duration = 14 [deprecated = true];
+
+  int64 channel_minimum_fee_msat = 15 [deprecated = true];
+  repeated OpeningFeeParams opening_fee_params_menu = 16;
```

Comment:
The mobile app only needs the short validity. MIght be a good idea to only expose the open channel fee for short validity here. Otherwise we will need to duplicate the logic of choosing the short validity in the flutter side as well.
What do you think @JssDWt ?

Reply:
Yeah, I wasn't sure whether that's the only place where LspInfo calls are used. Whether there is also a fee shown for a swap somewhere I mean? If that's the only place, that definitely a good solution.

Reply:
Actually, the same message is used by the swapper to get LSP info. I can add an additional field `cheapest_opening_fee_params`, so the flutter side can use that. What do you think @roeierez?